libs/design/guides/theming.md

```diff
@@ -0,0 +1,195 @@
+# Theming
+Daffodil's extensible theming architecture allows you to customize Daffodil Design components to match your brand or product's visual style using a set of universal variables, eliminating the need for individual component modifications.
```

Comment:
`Daffodil's extensible theming architecture allows you to customize our components`